internal/generate/teamcity/README.md

```diff
@@ -0,0 +1,26 @@
+# TeamCity Acceptance Test Configuration Generator
```

Comment:
This is a great idea (readme) in each generator. I've added some but I probably need to go back and document better.